orchestrator-salt/src/main/resources/salt/salt/cloudera/agent/init.sls

```diff
@@ -24,7 +24,7 @@ install-cloudera-manager-agent:
 
 {% endif %}
 
-{%- if not salt['pkg.version']('python-psycopg2') and not salt['pkg.version']('python2-psycopg2') %}
+{%- if not salt['pkg.version']('python-psycopg2') and not salt['pkg.version']('python2-psycopg2') and not salt['pkg.version']('python38-psycopg2') %}
```

Comment:
This is quite fragile, if RHEL8 moves to a newer version from python 3.8 we might install it back. Can't we just change the condition to check whether the OS version is 8 or 8+?

Reply:
This is not specific to RHEL 8 - sadly, package names are different based on various Python + OS combinations. Later on we'll have another round here with CentOS 7 +  Python 3.8.

Ultimately this is just a quick and dirty solution and what @lacikaaa has suggested above is much better, especially because according to CM Devs, CM won't even need this lib in the future. On the other hand, I'm afraid of disabling this for everything except ycloud because we need to keep backward compatibility with older versions like 7.2.6 as well. My idea would be to leave this as it is for now and revisit this topic with his solution as soon as we drop support for older CM / Runtime versions. 

(Extra info: On RHEL8 Python 3.6 is the current baseline and 3.8 is just a necessary upgrade agreed upon by different Cloudera teams. Apparently RedHat will skip Python 3.9 on RHEL8 as they made it default for RHEL 9 only. For Cloudera, Python 3.8 is the new big thing and I can't foresee proceeding to 3.9 anytime soon. All in all, I don't expect this to be a problem before RHEL 9)